hnginternship5/kidstories-android
# IMPLEMENTED USER CREATE STORY

### TITLE OF PR

User Create Stories
### PURPOSE OF PR

User should be able to add their own story
### FEATURES IMPLEMENTED

Add Story
### HOW TO TEST

When a user clicks on the add story button the user should able to Input the Title, upload Image, Input Content of the story and also post the story to the server.
### TRELLO BOARD  STORY LINK

https://trello.com/c/rNfmKHGR/2-user-create-stories
